orchestra/static/orchestra/timing/timecard/controller.js

```diff
@@ -74,7 +75,16 @@
           return;
         }
         entry.description = entry.editData.description;
-        entry.time_worked = moment.duration(entry.editData.timeWorked);
+
+        // Only truncate seconds worked if the user has changed things
+        if (!angular.equals(entry.editData.timeWorked, entry.time_worked.componentize())) {
+          entry.time_worked = moment.duration(entry.editData.timeWorked);
+        }
+
+        if (!entry.editData.date.isSame(entry.date)) {
```

Comment:
When I create a new entry, it doesn't auto-fill the date, so isSame throws a null exception